src/components/sectionList.re

```diff
@@ -12,7 +12,8 @@ and jsRenderBag('item) = {
   "index": int,
   "section": jsSection('item),
   "separators": {. "highlight": unit => unit, "unhighlight": unit => unit}
-};
+}
+and jsRenderAccessory('item) = {. "section": jsSection('item)};
```

Comment:
I'd prefer to have this the same as everything else so not using `and` but just defining it as another type.